staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/admission.go

```diff
@@ -139,10 +139,11 @@ func (a *QuotaAdmission) Validate(ctx context.Context, attr admission.Attributes
 	if attr.GetSubresource() != "" {
 		return nil
 	}
-	// ignore all operations that are not namespaced or creation of namespaces
-	if attr.GetNamespace() == "" || isNamespaceCreation(attr) {
-		return nil
-	}
+	// kcp edit: allow quota of cluster-scoped resources, so don't ignore them
```

Comment:
Why not just delete?

Reply:
It seemed better to keep it to indicate we were choosing to disable the check vs. an "accidental" deletion... ?